godal.go

```diff
@@ -3898,6 +3898,15 @@ func RegisterVSIHandler(prefix string, handler KeySizerReaderAt, opts ...VSIHand
 	return nil
 }
 
+// HasVSIHandler returns true if a VSIHandler is registered for this prefix
+func HasVSIHandler(prefix string) bool {
+	if handlers == nil {
+		return false
+	}
+	_, ok := handlers[prefix]
+	return ok
```

Comment:
```suggestion
	return handlers != nil && handlers[prefix] != nil
```
For better coverage points :)